### ISSUE TITLE
Dependency mismatch?

### ISSUE THREAD

**Issue by robopoc** [kaiko-ai/spark-dicom#36]
Running 

```
display(spark.read.format("dicomFile")
  .option("recursiveFileLookup", true)
  .option("includeContent", true)
  .load("/mnt/tcia/gcs-public-data--healthcare-tcia-apollo/"))
```

results in `NoSuchMethodError: org.apache.hadoop.fs.FSDataInputStream.readAllBytes()[B`

likely due to dependency mismatch.

**Comment by robopoc** [comment]
libraries used: 

org.dcm4che:dcm4che-core:5.24.2
org.dcm4che:dcm4che-imageio:5.24.2
spark_dicom_0_0_4_SNAPSHOT.jar

**Comment by robopoc** [comment]
using           
`val bytes = IOUtils.toByteArray(fileStream)
`
instead

**Comment by robopoc** [comment]
Not sure whether the above fixed it. I now get corrupted binary image data.

**Comment by GuillaumeDesforges** [comment]
Closed by 055e943872a52df86fa0fa7e7b6dee72be6057d5
